packages/mdc-chips/chip/foundation.js

```diff
@@ -107,6 +112,25 @@ class MDCChipFoundation extends MDCFoundation {
     }
   }
 
+  /**
+   * Handles a transition end event on the root element.
+   * This is a proxy for handling a transition end event on the leading icon or checkmark,
+   * since the transition end event bubbles.
+   * @param {!Event} evt
+   */
+  handleTransitionEnd_(evt) {
+    if (!(evt.propertyName === 'opacity')) {
+      return;
+    }
+    if (this.adapter_.eventTargetHasClass(evt.target, cssClasses.LEADING_ICON) &&
+        this.adapter_.hasClass(cssClasses.SELECTED)) {
+      this.adapter_.addClassToLeadingIcon(cssClasses.HIDDEN_LEADING_ICON);
+    } else if (this.adapter_.eventTargetHasClass(evt.target, cssClasses.CHECKMARK) &&
+              !this.adapter_.hasClass(cssClasses.SELECTED)) {
```

Comment:
If you're trying to line this line up after the opening paren, this needs one more space